core/src/main/java/com/airbnb/aerosolve/core/util/SupportVector.java

```diff
@@ -0,0 +1,90 @@
+package com.airbnb.aerosolve.core.util;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.Random;
+
+import com.airbnb.aerosolve.core.FunctionForm;
+import com.airbnb.aerosolve.core.ModelRecord;
+
+public class SupportVector implements Serializable {
```

Comment:
It seems `SupportVector` can also extend `AbstractFuction`, then RBF sv and  ARC_COSINE sv can be subclasses